101-episerver-in-azure/azuredeploy.json

```diff
@@ -0,0 +1,349 @@
+            "displayName":"StorageAccount"
+         },
+         "properties":{
+            "accountType":"[parameters('StorageAccountType')]"
+         }
+      },
+      {
+         "name":"[parameters('sqlserverName')]",
+         "type":"Microsoft.Sql/servers",
+         "location":"[resourceGroup().location]",
+         "apiVersion":"2014-04-01-preview",
+         "dependsOn":[
+
+         ],
+         "tags":{
+            "displayName":"SQL-Server"
+         },
+         "properties":{
+            "administratorLogin":"[parameters('sqlserverAdminLogin')]",
+            "administratorLoginPassword":"[parameters('sqlserverAdminLoginPassword')]",
+            "version":"12.0"
+         },
+         "resources":[
+            {
+               "name":"AllowAllWindowsAzureIps",
+               "type":"firewallrules",
+               "location":"[resourceGroup().location]",
+               "apiVersion":"2014-04-01-preview",
+               "dependsOn":[
+                  "[concat('Microsoft.Sql/servers/', parameters('sqlserverName'))]"
+               ],
+               "properties":{
+                  "startIpAddress":"0.0.0.0",
+                  "endIpAddress":"0.0.0.0"
+               }
+            },
+            {
+               "name":"[parameters('SQL-DatabaseName')]",
+               "type":"databases",
+               "location":"[resourceGroup().location]",
+               "apiVersion":"2014-04-01-preview",
+               "dependsOn":[
+                  "[parameters('sqlserverName')]"
+               ],
+               "tags":{
+                  "displayName":"SQL-Database"
+               },
+               "properties":{
+                  "collation":"[parameters('SQL-DatabaseCollation')]",
+                  "edition":"[parameters('SQL-DatabaseEdition')]",
+                  "maxSizeBytes":"1073741824",
+                  "requestedServiceObjectiveName":"[parameters('SQL-DatabaseRequestedServiceObjectiveName')]"
+               }
+            }
+         ]
+      },
+      {
+         "apiVersion":"2015-08-01",
+         "name":"[parameters('serviceBusNamespaceName')]",
+         "type":"Microsoft.ServiceBus/Namespaces",
+         "location":"[variables('location')]",
+         "kind":"Messaging",
+         "tags":{
+            "displayName":"ServiceBus"
+         },
+         "sku":{
+            "name":"[parameters('serviceBusSku')]",
+            "tier":"[parameters('serviceBusSku')]"
+         },
+         "properties":{
+
+         }
+      }
+   ],
+   "outputs":{
+      "NamespaceDefaultConnectionString":{
+         "type":"string",
+         "value":"[listkeys(variables('defaultAuthRuleResourceId'), variables('serviceBusApiVersion')).primaryConnectionString]"
+      },
+      "DefaultSharedAccessPolicyPrimaryKey":{
+         "type":"string",
+         "value":"[listkeys(variables('defaultAuthRuleResourceId'), variables('serviceBusApiVersion')).primaryKey]"
+      },
+      "DefaultSqlConnectionString":{
```

Comment:
Are we sure we want this as an output? Not sure if that's good security practice.